### PR TITLE
Bug 1218930 - test_clock_run_stopwatch_then_reset.py : AssertionError: u'00:01.58' == u'00:01.58 r=mwargers

### PR DESCRIPTION
**Pull request by cuff-links**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=13fc6d7c129f3683b72268f959972afb76a48526

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=00d4b34b10c947ca1a4a25b637e75f5dff5012c2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40a8d7476698c28322a93531c40e70ee387ed1e9

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (JohanLorenzo) [inline .idea/codeStyleSettings.xml]: Unnecessary file. Please remove it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=346358257aca436d0d3b93296484a7d963b2321f

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (silne30) [inline .idea/codeStyleSettings.xml]: @JohanLorenzo Thanks.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py [hunk 2]
```diff
@@ -31,12 +31,20 @@ def test_click_run_stopwatch_then_reset(self):
...
+
         time.sleep(0.2)
 
         self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
 
         self.assertNotEqual(first_time, stopwatch_view.current_time)
 
+        stopwatch_view.tap_pause()
+
+        time.sleep(0.2)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py]: I wonder what this sleep is. We should be able tap reset right after, shouldn't we?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py [hunk 3]
```diff
@@ -31,12 +31,20 @@ def test_click_run_stopwatch_then_reset(self):
...
+
         stopwatch_view.tap_reset()
 
+        time.sleep(0.2)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py]: Same question about the sleep.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=64f6492237b2e4193adfffa4d95eeed660477776

**Pull request merged**